recipes-core/sysvinit/sysvinit.bb

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) Cybertrust Japan Co., Ltd. 
+#
+# SPDX-License-Identifier: MIT
+#
+
+inherit dpkg
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI = " \
+apt://${PN} \
+file://custom-debian \
+"
+
+DEB_BUILD_PROFILES += "nocheck"
+
+PROVIDES += " \
+  sysvinit-core \
+  sysv-rc \
+  sysvinit-utils \
+  bootlogd \
+  busybox-syslogd \
+"
+
+do_move_debian_files() {
+    cp -r ${WORKDIR}/custom-debian/* ${S}/debian/
+}
+addtask move_debian_files after do_prepare_build before do_dpkg_build
```

Comment:
Please add following comments in this recipe:
- Why we need to create this recipe (instead of using binary pkg)
- Overview of difference for original bootclean.sh

Reply:
updated.

Reply:
> - Overview of difference for original bootclean.sh
I feel like there are not enough comments on this point of view. Could you please be more specific about the differences from the original script?